src/profile-logic/marker-data.js

```diff
@@ -64,7 +64,17 @@ export function deriveJankMarkers(
   let lastTimestamp: number = 0;
   const jankInstances = [];
   for (let i = 0; i < samples.length; i++) {
-    const currentResponsiveness = samples.responsiveness[i];
+    let currentResponsiveness;
+    if (samples.eventDelay) {
+      currentResponsiveness = samples.eventDelay[i];
+    } else if (samples.responsiveness) {
+      currentResponsiveness = samples.responsiveness[i];
+    } else {
+      throw new Error(
+        'There is no eventDelay or responsiveness array in the samples table!'
```

Comment:
I could just go with `if (samples.eventDelay) {} else {}` but wanted to check the third option for each case.

Also instead of throwing something, I thought about using `assertExhaustiveCheck` but it accepts only one value as parameter, which we have two in that case(`samples.eventDelay` and `samples.responsiveness`)

Reply:
Optional: I think I'd prefer to ignore if we don't put it in there, that way it's truly optional. I'll point out the other `throw`s below as well, just in case.